runtime/astar/src/precompiles.rs

```diff
@@ -29,6 +29,12 @@ pub const ASSET_PRECOMPILE_ADDRESS_PREFIX: &[u8] = &[255u8; 4];
 #[derive(Debug, Clone, Copy)]
 pub struct AstarNetworkPrecompiles<R, C>(PhantomData<(R, C)>);
 
+impl<R, C> Default for AstarNetworkPrecompiles<R, C> {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
This seems redundant, what was the warning?

Also, can `Default` just be added under `derive`?

Reply:
Lint error was
https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default
<img width="882" alt="Screen Shot 2022-07-07 at 16 37 28" src="https://user-images.githubusercontent.com/28953860/177718461-6b392a6e-c177-4940-89da-652147666877.png">